pkgs/build-support/build-dotnet-module/default.nix

```
@@ -102,63 +102,71 @@ let
 
       export HOME=$(mktemp -d)
 
-      dotnet restore "$projectFile" \
-        ${lib.optionalString (!enableParallelBuilding) "--disable-parallel"} \
-        -p:ContinuousIntegrationBuild=true \
-        -p:Deterministic=true \
-        --source "${nuget-source}/lib" \
-        "''${dotnetRestoreFlags[@]}" \
-        "''${dotnetFlags[@]}"
+      for project in ''${projectFile[@]}; do
```

Does this work :thinking: ?
```suggestion
      for project in "''${projectFile[@]}"; do
```

---

That's what I tried at first at well, but it doesn't. It doesn't interate over the array. Instead it will run once, with `$project` containing the entire array.